README.md

```diff
@@ -20,10 +23,16 @@ DC/OS Docker is designed to optimize developer cycle time. For a more production
 - A recent kernel that supports Overlay FS
 - git
 
-### Mac
+### macOS (Experimental)
 
-- VirtualBox 5.1.18
-- Vagrant 1.9.3
+- [Docker for Mac](https://docs.docker.com/docker-for-mac/) (17.03.1 tested).
+  See "Graphdriver/Storage driver"
+- git
+
+### Vagrant on either platform
```

Comment:
I will but yes this is another reason why `ReST` is so much better than MD for these purposes - you choose the anchor.